evopy/evopy.py

```diff
@@ -54,11 +59,18 @@ def run(self):
         return best.genotype
 
     def _init_population(self):
+        if self.strategy == Strategy.SINGLE_VARIANCE:
+            strategy_parameters = [np.random.randn()]
+        elif self.strategy == Strategy.MULTIPLE_VARIANCE:
+            strategy_parameters = np.random.randn(self.individual_length)
+        else:
```

Comment:
Is this the case for full variance? If so, please make it an elif to check that that strategy was used